UI/UnoMaterialSample/UnoMaterialSample.Droid/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="UnoMaterialSample" android:versionCode="1" android:versionName="1.0">
-  <uses-sdk android:minSdkVersion="16" android:targetSdkVersion="29" />
-  <application android:label="UnoMaterialSample"></application>
-</manifest>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="UnoMaterialSample.UnoMaterialSample" android:versionCode="1" android:versionName="1.0" android:installLocation="auto">
```

Comment:
Is this package name change needed?

Reply:
It was automatic by the VS. I just changed the target to 31.